GameOfLife/Board.php

```diff
@@ -158,54 +149,49 @@ function calculateNextGeneration()
                         $newValue = 1;
                     }
                 }
-
                 $newBoard[$widthId][$heightId] = $newValue;
             }
         }
         // Sezt an letzer stelle ein neues array
         $this->previousGenerations[] = $this->board;
         $this->board = $newBoard;
-
     }
 
     /**
-     * @meth
-     *
      *Executes the board and puts - and * instead of 0 and 1
      */
     function printBoard()
     {
         for ($y = 0; $y < $this->height; ++$y) {
             for ($x = 0; $x < $this->width; ++$x) {
                 $character = "-";
-                if ($this->board[$x][$y] == 1) $character = "*";
+                if ($this->board[$y][$x] == 1) $character = "*";
                 echo " $character ";
             }
             echo "\n";
         }
     }
 
     /**
-     * @meth
-     *
-     *Checks if the generations repeat
+     * Checks if the generations repeat.
      */
-    function shouldFinish()
+    function shouldFinish(): int
```

Comment:
Glad you entered return values! :+1: But the correct return value in this case would be `bool`.